internal/abci/abci.go

```diff
@@ -10,6 +10,7 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/kwilteam/kwil-db/common/sql"
```

Comment:
Previously, abci wasn't really made aware to the database, and txapp contained all of the logic and transaction management. Breaking that here feels pretty weird IMO. Not sure if I have a concrete suggestion on how to fix it.